appmap/_implementation/event.py

```diff
@@ -59,10 +59,10 @@ def display_string(val):
     value = None
     if Env.current.display_params:
         try:
-            value = str(val)
+            value = repr(val)
         except Exception:  # pylint: disable=broad-except
             try:
-                value = repr(val)
+                value = str(val)
```

Comment:
I wonder if we even need this? If `repr` didn't work, what are the chances that `str` will?

Reply:
Good point, I'll remove it.